### ISSUE TITLE
Rescan fails

### ISSUE THREAD

**Issue by bar17** [chris-belcher/electrum-personal-server#129]
Hello, I recently had EPS working appropriately but the rescan script is no longer working.

I am using Windows, Bitcoin Core 018.0, EPS 0.17, Electrum 3.3.6.

I originally had it working with a few ypubs and the watch only addresses included sats from all ypubs. Now, I added a few more ypubs and a zpub and the rescan script will not run. The CLI simply pops up, quickly displays some text, and disappears. There is no text added to the ..."\AppData\Local\Temp\elecrumpersonalserver.log" to help with trouble shooting after attempting to run the re-scan script. I also tried changing from INFO to DEBUG. No change.

I deleted the log, created a new BitcoinCore wallet, re-ran the config.ini file with electrum-personal-server.exe to load the addresses and rescan still will not work as it did before (asking me to give it a date to rescan from). This time it is only using ypubs and I left he zpub out.

If I search for addresses in BitcoinCore it finds them as watch only. So the wallet has successfully imported the addresses.

Any help on how to troubleshoot this is appreciated.

**Comment by chris-belcher** [comment]
In the same directory as `electrum-personal-server.exe`, hold the shift key and right-click somewhere in the file explorer, then choose "Open command window here". That should open a terminal window, then type `electrum-personal-server-rescan config.ini` and press enter. That should run the rescan script in the terminal and allow you to see the error message. Change `config.ini` to whatever is the name of your configuration file.

**Comment by bar17** [comment]
This worked immediately without error. Further, the text command line made me realize my error. I started to drag the rescan.bat file on top of the .exe file instead of the config.ini file. It was a stupid mistake, but I committed to it over and over after previously doing it correctly for some reason!

Perhaps adding command line instructions to the tutorial would be helpful, although this doesn't seem like a priority.

Thank you!
